src/interfaces/coral_web/src/components/Shared/Icon/Icon.tsx

```diff
@@ -77,6 +77,8 @@ export const Icons = [
   'warning',
   'web',
   'compass',
+  'wrench',
+  'share',
```

Comment:
i think we already have `link`?

Reply:
jk i guess this is `share` 
<img width="41" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/098d0880-53e6-4faf-aff6-bd53f63ee937">